sortinghat/cli/client/schema.py

```diff
@@ -183,7 +183,8 @@ class CountryType(sgqlc.types.Type):
     code = sgqlc.types.Field(sgqlc.types.non_null(String), graphql_name='code')
```

Comment:
We should exclude this file from the checks. It's going to keep failing because it is generated automatically.

Reply:
Okay. :+1:
The flake8 configuration file is missing anyway. I think we can add this file to the `exclude` list.